feature/interface/ipv6_ND/otg_tests/disable_ipv6_nd_ra_test/README.md

```diff
@@ -0,0 +1,51 @@
+# RT-5.x: Interface Loopback mode
+[TODO] assign test number 
+
+## Summary
+
+Validate IPv6 ND Router Advertisement (RA) could be completely disabled.
+
+## Procedure
+*   Connect DUT porr-1 to OTG port-1
+*   Configure IPv6 address on DUT port-1
+*   set IPv6 ND RA transmission interval on DUT port-1 to 5 seconds
+*   disable IPv6 ND RA transmission on DUT port-1
```

Comment:
first set, then disable? Why to not just disable right away? 
also, start with capitals pls to be consistent with first two lines